docs/api/index.md

```diff
@@ -0,0 +1,2 @@
+# PLACEHOLDER

```

Comment:
i'd appreciate if we could do without placeholders so we can add actual contents whether the PR is merged or not

Reply:
removed placeholders, gonna do something else and then i will do workflow